internal/models/ask_questions_model_test.go

```
@@ -0,0 +1,154 @@
+	updatedModel, ok := newModel.(*AskQuestionsModel)
+	if !ok {
+		t.Fatal("Could not convert updated model back to AskQuestionsModel")
+	}
+
+	if updatedModel.width != 200 || updatedModel.height != 100 {
+		t.Errorf("Expected dimensions to be (200, 100), got (%d, %d)",
+			updatedModel.width, updatedModel.height)
+	}
+
+	// Test escape key
+	newModel, cmd := model.Update(tea.KeyMsg{Type: tea.KeyEsc})
```

golangci-lint (1.64.8)

ineffectual assignment to newModel

(ineffassign)